roles/pcp/templates/target-host

```diff
@@ -0,0 +1,5 @@
+# {{ ansible_managed }}
+{{ lab_domain }}
```

Comment:
doesn't seem like this should be here?  pmmgr(1) says target-host contains "one or more lines containing host specs"